tests/testthat/test-derive_vars_query.R

```diff
@@ -150,7 +156,8 @@ test_that("Derive decides between TERM_NAME and TERM_ID based on the type of the
 })
 
 
-test_that("assert_valid_queries checks VAR_PREFIX values", {
+## Test 6: assert_valid_queries checks ----
+test_that("derive_vars_query Test 6: assert_valid_queries checks", {
```

Comment:
The test tests `assert_valid_queries()`. Thus the test description should start with "assert_valid_queries".